bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -30,11 +30,11 @@ export default function userLeaving(credentials, userId) {
       'user-not-found', `You need a valid user to be able to toggle audio`);
   }
 
-  if (User.user.connection_status === OFFLINE_CONNECTION_STATUS) {
+  if (User.connection_status === OFFLINE_CONNECTION_STATUS) {
```

Comment:
There is a nested object in a JSON, right. And all modifiers are set to update `'user.connection_status'`.
I was just switching between branches/versions of Meteor and accidentally got those 3 exceptions. 
No idea what exactly caused that exception - I'll revert it to whatever it was before and let's see if we can reproduce it later.